x-pack/legacy/plugins/fleet/doc/agent_api_checkin.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[fleet-agent-checkin]]
+=== Fleet agent checkin api
+++++
+<titleabbrev>Enroll a new agent</titleabbrev>
+++++
+
+[[fleet-agent-checkin-request]]
+==== Request
+
+`POST /api/fleet/agents/{agentId}/checkin` 
+
+==== Headers
+
+`kbn-fleet-access-token`::
+  (Requered, string) A fleet agent access token.
+
+[[fleet-agent-checkin-request-request-codes]]
+
+==== Request body
+
+`events`::
+  (Required, array) An array of events with the properties `type`, `subtype`, `message`, `timestamp`, `payload`, `data`.
```

Comment:
I would add an `and` in there. So it would look like:

...`payload`, and `data`.